test/bigquery/table.js

```diff
@@ -281,6 +279,18 @@ describe('BigQuery/Table', function() {
       table.createWriteStream(fileType).emit('writing');
     });
 
+    it('should format a schema', function(done) {
+      var expectedSchema = Table.createSchemaFromString_(SCHEMA_STRING);
```

Comment:
Should we be stubbing `createSchemaFromString_` to return fake data and assert against said fake data instead?